docs/guides/setup/configuring-metadata-stores/how-to-configure-a-validation-result-store-in-amazon-s3.md

```diff
@@ -2,4 +2,103 @@
+
+<Prerequisites>
+
+- Configured a [Data Context](../../../tutorials/getting-started/initialize-a-data-context.md).
+- Configured an [Expectations Suite](../../../tutorials/getting-started/create-your-first-expectations.md).
+- Configured a [Checkpoint](../../../tutorials/getting-started/validate-your-data.md).
+- Installed [boto3](https://github.com/boto/boto3) in your local environment.
+- Identified the S3 bucket and prefix where Validation results will be stored.
+
+</Prerequisites>
+
+Steps
+-----
+
+1. **Configure** [boto3](https://github.com/boto/boto3) **to connect to the Amazon S3 bucket where Validation results will be stored.**
+
+    Instructions on how to set up [boto3](https://github.com/boto/boto3) with AWS can be found at boto3's [documentation site](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html).
+
+2. **Identify your Data Context Validations Store**
+
+   Look for the following section in your Data Context's ``great_expectations.yml`` file:
+
+   ```yaml
+      validations_store_name: validations_store
+
+      stores:
+         validations_store:
+             class_name: ValidationsStore
+             store_backend:
+                 class_name: TupleFilesystemStoreBackend
+                 base_directory: uncommitted/validations/
+   ```
+
+   The configuration file tells Great Expectations to look for Validations in a store called ``validations_store``. It also creates a ``ValidationsStore`` called ``validations_store`` that is backed by a Filesystem and will store validations under the ``base_directory`` ``uncommitted/validations`` (the default).
+
+
+3. **Update your configuration file to include a new store for Validation results on S3.**
+
+    In the example below, the new store's name is set to ``validations_S3_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``TupleS3StoreBackend``, ``bucket`` will be set to the address of your S3 bucket, and ``prefix`` will be set to the folder in your S3 bucket where Validation results will be located.
+
+
+:::caution
+
+   If you are also storing Expectations in S3 ([How to configure an Expectation store to use Amazon S3](./how-to-configure-an-expectation-store-in-amazon-s3)), or DataDocs in S3 ([How to host and share Data Docs on Amazon S3](../configuring-data-docs/how-to-host-and-share-data-docs-on-amazon-s3)), then please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
+
+:::
+
+   ```yaml
+   validations_store_name: validations_S3_store
+
+   stores:
+      validations_S3_store:
+          class_name: ValidationsStore
+          store_backend:
+              class_name: TupleS3StoreBackend
+              bucket: '<your_s3_bucket_name>'
+              prefix: '<your_s3_bucket_folder_name>'
+   ```
```

Comment:
This block is not indented but the other ones are. How should we approach these?